auth/resources/authserver.properties

```diff
@@ -47,7 +47,8 @@ defaultEndpoint=127.0.0.1
 s3Endpoints=s3-us-west-2.seagate.com,s3-us.seagate.com,s3-europe.seagate.com,s3-asia.seagate.com,s3.seagate.com
 #Auth Server to S3 connection will be over http or https as specified below.
 enableHttpsToS3=false
-
+# Command to fetch LDAP password from Cortx Security tools
```

Comment:
Remove comment, we might need see if this will be secure, as we are specifying complete commad to fetch password here

Reply:
done